src/foss-update.php

```diff
@@ -35,6 +35,16 @@ public function patch()
     }
 }
 
+class FOSSPatch_Favicon extends FOSSPatchAbstract
```

Comment:
Naming of the class should be FOSSpatch_26 in this case 
See
https://github.com/boxbilling/boxbilling/blob/master/src/bb-update.php

Also it should be before FOSSPatch_25